### PR TITLE
Redirect versioned hub docs to the latest version, as the historical …

### PR DESCRIPTION
**Pull request by SvenDowideit**
…ones aren't useful

Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

and this one for the current release.

### CODE REVIEW

File: redirects.csv [hunk 1]
```diff
@@ -10,7 +9,7 @@ extend/,engine/extend/
 installation/,engine/installation/
 introduction/,engine/introduction/
 misc/,engine/misc/
-project/,opensource/project/
+project/,engine/project/
```
Comment (moxiegirl) [inline redirects.csv]: This is wrong.  it is now in `opensource`

Comment (SvenDowideit) [inline redirects.csv]: ah, cool - I'll fix that in the master PR too

File: redirects.csv [hunk 2]
```diff
@@ -2,7 +2,6 @@
 # so "en/latest/," removes the "en/latest/" prefix from any incoming URL and redirects it to something on the root
 # from,to
 # Oct 2015: moved docker/docker docs from root to /engine/
-articles/networking,engine/userguide/networking
```
Comment (moxiegirl) [inline redirects.csv]: Why are you removing this?

**Comment by moxiegirl** [comment]
LGTM

**Pull request merged**